src/components/ui/InputTagAdd/index.tsx

```diff
@@ -0,0 +1,27 @@
+'use client';
+
+import { clsx } from 'clsx';
+import IconAdd from '@/assets/icons/icon-add.svg';
+
+type InputTagAddProps = {
+  onClick: () => void;
+  className?: string;
+};
+
+export const InputTagAdd = ({ onClick, className }: InputTagAddProps) => {
+  return (
+    <button
+      onClick={onClick}
+      className={clsx(
+        'h-[40px] w-[40px]',
+        'flex items-center justify-center',
+        'border-layout-grey6 rounded-[6px] border',
+        'hover:bg-layout-grey1 bg-white',
+        'transition-colors duration-150',
+        className,
+      )}
+    >
+      <IconAdd className="h-[14px] w-[14px] overflow-visible" />
```

Comment:
p1) 얘를 억지로 사이즈 조절하려다 보니까 중심이 안 맞는 거 같아요! 아이콘은 이미 14px라서 
```suggestion
      <IconAdd  />
```

만 적어줘도 될 것 같습니다

Reply:
반영했습니다